net.worldofpadman.WoP.yaml

```diff
@@ -0,0 +1,56 @@
+app-id: net.worldofpadman.WoP
+runtime: org.freedesktop.Platform
+runtime-version: 22.08
+sdk: org.freedesktop.Sdk
+command: worldofpadman
+finish-args:
+  # Controller support
+  - --device=all
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --persist=.padman
+cleanup:
+  - /man
+  - /include
+  - /share/man
+  - /doc
+  - '*.a'
+  - '*.la'
+modules:
+  - name: wop
+    buildsystem: simple
+    sources:
+      - type: archive
+        url: https://github.com/PadWorld-Entertainment/worldofpadman/archive/refs/tags/v1.6.2.tar.gz
+        sha256: dab22da3b9b8602a32425463fcfb22b443ef68c47e90d6f916a3a07fe17277c5
+      - type: file
+        path: net.worldofpadman.WoP.desktop
+      - type: file
+        path: net.worldofpadman.WoP.svg
+      - type: file
+        path: net.worldofpadman.WoP.metainfo.xml
+      - type: script
+        commands:
+          - exec /app/share/games/wop/wop.x86_64 "$@"
+        dest-filename: worldofpadman
+    build-commands:
+      - mkdir build && cd build && cmake .. -DCMAKE_BUILD_TYPE=Release && cmake --build .
```

Comment:
buildsystem should be cmake-ninja then. (or cmake as a fallback)

Reply:
It does not work properly that way. At least the installation has to be done manually - as I mentioned, the official archive is a self-contained zip. I may write a downstream patch for this later, but it works fine this way for now.

Reply:
you can skip the `make install`.

Reply:
Can you please tell me how exactly? I most likely already did it on my other Flatpaks, but forgot it and I am a little more slower and more dumb than usual now since all I got for Xmas is COVID. :-/

Thanks!

Reply:
Done.